ocis-pkg/config/config.go

```diff
@@ -73,7 +73,7 @@ type Config struct {
 	Runtime           Runtime              `yaml:"runtime"`
 
 	Accounts          *accounts.Config        `yaml:"accounts"`
-	AppProvider       *appprovider.Config     `yaml:"appprovider"`
+	AppProvider       *appProvider.Config     `yaml:"app-provider"`
```

Comment:
we do the `-` to `_` change for all yaml tags in a separate PR?

Reply:
would be fine for me